examples/react-router/src/pages/Login.tsx

```diff
@@ -19,15 +21,16 @@ const LoginPage = withRouter(({ location }) => {
   }, [keycloak])
 
   if (keycloak?.authenticated)
-    return <Redirect to={currentLocationState?.from as string} />
+    return <Redirect to={currentLocationSte?.from as string} />
```

Comment:
I suspect there's a typo here and this would cause the app not to work since `currentLocationSte` is always `undefined`